songbird/queue.py

```diff
@@ -44,11 +44,12 @@ class Queue(list):
     def __init__(
         self,
         driver: Driver,
-        on_next: Callable[[Driver, Any], Awaitable[None]] = None,
-        on_fail: Callable[[Driver, Any], Awaitable[None]] = None
+        on_next: typing.Callable[[Driver, typing.Any], typing.Awaitable[None]] | None = None,
+        on_fail: typing.Callable[[Driver, typing.Any], typing.Awaitable[None]] | None = None
     ) -> None:
+        super().__init__()
         self.driver = extract_driver(driver)
-        self.track_handle: Optional[TrackHandle] = None
+        self.track_handle: TrackHandle | None = None
```

Comment:
Pipes aren't used for Union + Optional in this project

Reply:
It looks much prettier, python 3.11 style

Reply:
https://www.python.org/dev/peps/pep-0563/